commit c635e53e4c740a463463c844487480739529a30f
Author: Matthias Becker <ma.becker@sap.com>
Date:   Wed Apr 12 10:04:13 2017 +0200

    Bug 514863: Use png files in org.eclipse.equinox.p2.ui
    
    Change-Id: Ia891112782c251e903149d5e3db8d873f3b1b6d9
    Signed-off-by: Matthias Becker <ma.becker@sap.com>

-	-	bundles/org.eclipse.equinox.p2.ui.admin.rcp/icons/view/provision.png
-	-	bundles/org.eclipse.equinox.p2.ui.admin.rcp/icons/view/provision@2x.png
-	-	bundles/org.eclipse.equinox.p2.ui.admin.rcp/icons/view/pview.png
-	-	bundles/org.eclipse.equinox.p2.ui.admin.rcp/icons/view/pview@2x.png
6	6	bundles/org.eclipse.equinox.p2.ui.admin.rcp/plugin.xml
-	-	bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/artifact_repo_obj.gif
-	-	bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/artifact_repo_obj.png
-	-	bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/artifact_repo_obj@2x.png
-	-	bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/iu_obj.gif
-	-	bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/iu_obj.png
-	-	bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/iu_obj@2x.png
-	-	bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/metadata_repo_obj.gif
-	-	bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/metadata_repo_obj.png
-	-	bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/metadata_repo_obj@2x.png
-	-	bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/profile_obj.gif
-	-	bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/profile_obj.png
-	-	bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/profile_obj@2x.png
-	-	bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/uninstalled_iu.gif
-	-	bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/uninstalled_iu.png
-	-	bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/uninstalled_iu@2x.png
-	-	bundles/org.eclipse.equinox.p2.ui.admin/icons/view/provision.png
-	-	bundles/org.eclipse.equinox.p2.ui.admin/icons/view/provision@2x.png
6	6	bundles/org.eclipse.equinox.p2.ui.admin/plugin.xml
-	-	bundles/org.eclipse.equinox.p2.ui.discovery/icons/etool16/find.png
-	-	bundles/org.eclipse.equinox.p2.ui.discovery/icons/etool16/find@2x.png
-	-	bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/iu_disabled_obj.gif
-	-	bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/iu_disabled_obj.png
-	-	bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/iu_disabled_obj@2x.png
-	-	bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/iu_obj.gif
-	-	bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/iu_obj.png
-	-	bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/iu_obj@2x.png
-	-	bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/iu_update_obj.gif
-	-	bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/iu_update_obj.png
-	-	bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/iu_update_obj@2x.png
-	-	bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/message_info.gif
-	-	bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/message_info.png
-	-	bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/message_info@2x.png
-	-	bundles/org.eclipse.equinox.p2.ui.discovery/icons/ovr32/message_warning.png
-	-	bundles/org.eclipse.equinox.p2.ui.discovery/icons/ovr32/message_warning@2x.png
-	-	bundles/org.eclipse.equinox.p2.ui.discovery/icons/wizban/banner-discovery.png
-	-	bundles/org.eclipse.equinox.p2.ui.discovery/icons/wizban/banner-discovery@2x.png
5	5	bundles/org.eclipse.equinox.p2.ui.discovery/src/org/eclipse/equinox/internal/p2/ui/discovery/DiscoveryImages.java
-	-	bundles/org.eclipse.equinox.p2.ui.importexport/icons/install_wiz.gif
-	-	bundles/org.eclipse.equinox.p2.ui.importexport/icons/wizban/install_wiz.png
-	-	bundles/org.eclipse.equinox.p2.ui.importexport/icons/wizban/install_wiz@2x.png
4	4	bundles/org.eclipse.equinox.p2.ui.importexport/plugin.xml
3	3	bundles/org.eclipse.equinox.p2.ui.importexport/src/org/eclipse/equinox/internal/p2/importexport/internal/wizard/ExportWizard.java
62	62	bundles/org.eclipse.equinox.p2.ui.importexport/src/org/eclipse/equinox/internal/p2/importexport/internal/wizard/ImportFromInstallationWizard.java
1	1	bundles/org.eclipse.equinox.p2.ui.importexport/src/org/eclipse/equinox/internal/p2/importexport/internal/wizard/ImportWizard.java
-	-	bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/close.gif
-	-	bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/close.png
-	-	bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/close@2x.png
-	-	bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/close_hot.gif
-	-	bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/close_hot.png
-	-	bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/close_hot@2x.png
-	-	bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/update.gif
-	-	bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/update.png
-	-	bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/update@2x.png
-	-	bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/update_problems.gif
-	-	bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/update_problems.png
-	-	bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/update_problems@2x.png
7	7	bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/src/org/eclipse/equinox/internal/p2/ui/sdk/scheduler/AutomaticUpdatePlugin.java
2	2	bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/src/org/eclipse/equinox/internal/p2/ui/sdk/scheduler/migration/MigrationWizard.java
-	-	bundles/org.eclipse.equinox.p2.ui.sdk/icons/obj/iu_obj.gif
-	-	bundles/org.eclipse.equinox.p2.ui.sdk/icons/obj/iu_obj.png
-	-	bundles/org.eclipse.equinox.p2.ui.sdk/icons/obj/iu_obj@2x.png
-	-	bundles/org.eclipse.equinox.p2.ui.sdk/icons/obj/iu_update_obj.gif
-	-	bundles/org.eclipse.equinox.p2.ui.sdk/icons/obj/iu_update_obj.png
-	-	bundles/org.eclipse.equinox.p2.ui.sdk/icons/obj/iu_update_obj@2x.png
-	-	bundles/org.eclipse.equinox.p2.ui.sdk/icons/obj/profile_obj.gif
-	-	bundles/org.eclipse.equinox.p2.ui.sdk/icons/obj/profile_obj.png
-	-	bundles/org.eclipse.equinox.p2.ui.sdk/icons/obj/profile_obj@2x.png
11	11	bundles/org.eclipse.equinox.p2.ui.sdk/plugin.xml
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/added_overlay.gif
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/artifact_repo_obj.gif
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/artifact_repo_obj.png
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/artifact_repo_obj@2x.png
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/category_obj.gif
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/category_obj.png
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/category_obj@2x.png
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_add.gif
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_add.png
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_add@2x.png
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_disabled_obj.gif
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_disabled_obj.png
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_disabled_obj@2x.png
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_disabled_patch_obj.gif
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_disabled_patch_obj.png
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_disabled_patch_obj@2x.png
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_downgraded.png
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_downgraded@2x.png
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_info.gif
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_info.png
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_info@2x.png
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_notadd.gif
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_notadd.png
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_notadd@2x.png
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_obj.gif
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_obj.png
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_obj@2x.png
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_patch_obj.gif
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_patch_obj.png
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_patch_obj@2x.png
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_remove.gif
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_remove.png
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_remove@2x.png
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_update_obj.gif
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_update_obj.png
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_update_obj@2x.png
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_upgraded.png
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_upgraded@2x.png
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/metadata_repo_obj.gif
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/metadata_repo_obj.png
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/metadata_repo_obj@2x.png
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/profile_obj.gif
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/profile_obj.png
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/profile_obj@2x.png
-	-	bundles/org.eclipse.equinox.p2.ui/icons/obj/removed_overlay.gif
-	-	bundles/org.eclipse.equinox.p2.ui/icons/ovr/added_overlay.png
-	-	bundles/org.eclipse.equinox.p2.ui/icons/ovr/added_overlay@2x.png
-	-	bundles/org.eclipse.equinox.p2.ui/icons/ovr/removed_overlay.png
-	-	bundles/org.eclipse.equinox.p2.ui/icons/ovr/removed_overlay@2x.png
-	-	bundles/org.eclipse.equinox.p2.ui/icons/wizban/install_wiz.gif
-	-	bundles/org.eclipse.equinox.p2.ui/icons/wizban/install_wiz.png
-	-	bundles/org.eclipse.equinox.p2.ui/icons/wizban/install_wiz@2x.png
-	-	bundles/org.eclipse.equinox.p2.ui/icons/wizban/revert_wiz.gif
-	-	bundles/org.eclipse.equinox.p2.ui/icons/wizban/revert_wiz.png
-	-	bundles/org.eclipse.equinox.p2.ui/icons/wizban/revert_wiz@2x.png
-	-	bundles/org.eclipse.equinox.p2.ui/icons/wizban/uninstall_wiz.gif
-	-	bundles/org.eclipse.equinox.p2.ui/icons/wizban/uninstall_wiz.png
-	-	bundles/org.eclipse.equinox.p2.ui/icons/wizban/uninstall_wiz@2x.png
-	-	bundles/org.eclipse.equinox.p2.ui/icons/wizban/update_wiz.gif
-	-	bundles/org.eclipse.equinox.p2.ui/icons/wizban/update_wiz.png
-	-	bundles/org.eclipse.equinox.p2.ui/icons/wizban/update_wiz@2x.png
24	24	bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/ProvUIImages.java
diff --git a/bundles/org.eclipse.equinox.p2.ui.admin.rcp/icons/view/provision.png b/bundles/org.eclipse.equinox.p2.ui.admin.rcp/icons/view/provision.png
new file mode 100644
index 0000000..bbd0d4a
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.admin.rcp/icons/view/provision.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.admin.rcp/icons/view/provision@2x.png b/bundles/org.eclipse.equinox.p2.ui.admin.rcp/icons/view/provision@2x.png
new file mode 100644
index 0000000..b540633
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.admin.rcp/icons/view/provision@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.admin.rcp/icons/view/pview.png b/bundles/org.eclipse.equinox.p2.ui.admin.rcp/icons/view/pview.png
new file mode 100644
index 0000000..2565e61
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.admin.rcp/icons/view/pview.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.admin.rcp/icons/view/pview@2x.png b/bundles/org.eclipse.equinox.p2.ui.admin.rcp/icons/view/pview@2x.png
new file mode 100644
index 0000000..cda7dc9
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.admin.rcp/icons/view/pview@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.admin.rcp/plugin.xml b/bundles/org.eclipse.equinox.p2.ui.admin.rcp/plugin.xml
index aa8349e..4095565 100644
--- a/bundles/org.eclipse.equinox.p2.ui.admin.rcp/plugin.xml
+++ b/bundles/org.eclipse.equinox.p2.ui.admin.rcp/plugin.xml
@@ -25,22 +25,22 @@
                value="%aboutText"/>
          <property
                name="windowImages"
-               value="platform:/plugin/org.eclipse.equinox.p2.ui.admin/icons/provision.gif"/>
-               
+               value="platform:/plugin/org.eclipse.equinox.p2.ui.admin/icons/view/provision.png"/>
+
     </product>
     </extension>
-    
+
        <extension
          point="org.eclipse.ui.views">
       <view
             name="%Views.Progress"
-            icon="$nl$/icons/pview.gif"
+            icon="$nl$/icons/view/pview.png"
             category="org.eclipse.ui"
             class="org.eclipse.ui.internal.progress.ProgressView"
             id="org.eclipse.ui.views.ProgressView">
       </view>
     </extension>
-    
+
          <extension
          point="org.eclipse.ui.perspectives">
       <perspective
@@ -49,5 +49,5 @@
             id="org.eclipse.equinox.p2.ui.admin.rcp.ProvisioningRCPPerspective">
       </perspective>
    </extension>
-   
+
 </plugin>
diff --git a/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/artifact_repo_obj.gif b/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/artifact_repo_obj.gif
deleted file mode 100644
index 0470e15..0000000
Binary files a/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/artifact_repo_obj.gif and /dev/null differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/artifact_repo_obj.png b/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/artifact_repo_obj.png
new file mode 100644
index 0000000..5a4b140
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/artifact_repo_obj.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/artifact_repo_obj@2x.png b/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/artifact_repo_obj@2x.png
new file mode 100644
index 0000000..de198ab
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/artifact_repo_obj@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/iu_obj.gif b/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/iu_obj.gif
deleted file mode 100644
index 8706680..0000000
Binary files a/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/iu_obj.gif and /dev/null differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/iu_obj.png b/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/iu_obj.png
new file mode 100644
index 0000000..684e82a
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/iu_obj.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/iu_obj@2x.png b/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/iu_obj@2x.png
new file mode 100644
index 0000000..d61d826
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/iu_obj@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/metadata_repo_obj.gif b/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/metadata_repo_obj.gif
deleted file mode 100644
index b8568f4..0000000
Binary files a/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/metadata_repo_obj.gif and /dev/null differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/metadata_repo_obj.png b/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/metadata_repo_obj.png
new file mode 100644
index 0000000..2cb962a
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/metadata_repo_obj.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/metadata_repo_obj@2x.png b/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/metadata_repo_obj@2x.png
new file mode 100644
index 0000000..3227b4f
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/metadata_repo_obj@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/profile_obj.gif b/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/profile_obj.gif
deleted file mode 100644
index c984fba..0000000
Binary files a/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/profile_obj.gif and /dev/null differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/profile_obj.png b/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/profile_obj.png
new file mode 100644
index 0000000..bbd0d4a
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/profile_obj.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/profile_obj@2x.png b/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/profile_obj@2x.png
new file mode 100644
index 0000000..b540633
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/profile_obj@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/uninstalled_iu.gif b/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/uninstalled_iu.gif
deleted file mode 100644
index f192b84..0000000
Binary files a/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/uninstalled_iu.gif and /dev/null differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/uninstalled_iu.png b/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/uninstalled_iu.png
new file mode 100644
index 0000000..2903413
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/uninstalled_iu.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/uninstalled_iu@2x.png b/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/uninstalled_iu@2x.png
new file mode 100644
index 0000000..fda2009
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.admin/icons/obj/uninstalled_iu@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.admin/icons/view/provision.png b/bundles/org.eclipse.equinox.p2.ui.admin/icons/view/provision.png
new file mode 100644
index 0000000..bbd0d4a
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.admin/icons/view/provision.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.admin/icons/view/provision@2x.png b/bundles/org.eclipse.equinox.p2.ui.admin/icons/view/provision@2x.png
new file mode 100644
index 0000000..b540633
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.admin/icons/view/provision@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.admin/plugin.xml b/bundles/org.eclipse.equinox.p2.ui.admin/plugin.xml
index 6aa8508..9c44a47 100644
--- a/bundles/org.eclipse.equinox.p2.ui.admin/plugin.xml
+++ b/bundles/org.eclipse.equinox.p2.ui.admin/plugin.xml
@@ -23,7 +23,7 @@
       </category>
       <view
             name="%MetadataRepositories.name"
-            icon="$nl$/icons/obj/metadata_repo_obj.gif"
+            icon="$nl$/icons/obj/metadata_repo_obj.png"
             category="org.eclipse.p2.ui.admin"
             class="org.eclipse.equinox.internal.p2.ui.admin.MetadataRepositoriesView"
             id="org.eclipse.p2.ui.admin.MetadataRepositoriesView">
@@ -33,7 +33,7 @@
          point="org.eclipse.ui.views">
      <view
             name="%ArtifactRepositories.name"
-            icon="$nl$/icons/obj/artifact_repo_obj.gif"
+            icon="$nl$/icons/obj/artifact_repo_obj.png"
             category="org.eclipse.p2.ui.admin"
             class="org.eclipse.equinox.internal.p2.ui.admin.ArtifactRepositoriesView"
             id="org.eclipse.p2.ui.admin.ArtifactRepositoriesView">
@@ -43,18 +43,18 @@
          point="org.eclipse.ui.views">
      <view
             name="%Profiles.name"
-            icon="$nl$/icons/obj/profile_obj.gif"
+            icon="$nl$/icons/obj/profile_obj.png"
             category="org.eclipse.p2.ui.admin"
             class="org.eclipse.equinox.internal.p2.ui.admin.ProfilesView"
             id="org.eclipse.p2.ui.admin.ProfilesView">
       </view>
    </extension>
-   
+
      <extension
          point="org.eclipse.ui.perspectives">
       <perspective
             name="%perspectiveName"
-            icon="$nl$/icons/provision.gif"
+            icon="$nl$/icons/view/provision.png"
             class="org.eclipse.equinox.internal.p2.ui.admin.ProvisioningPerspective"
             id="org.eclipse.equinox.p2.ui.admin.ProvisioningPerspective">
       </perspective>
@@ -95,7 +95,7 @@
              <adapt type="org.eclipse.equinox.p2.repository.IRepository"/>
            </enabledWhen>
       </page>
-      
+
       </extension>
 
 </plugin>
diff --git a/bundles/org.eclipse.equinox.p2.ui.discovery/icons/etool16/find.png b/bundles/org.eclipse.equinox.p2.ui.discovery/icons/etool16/find.png
index 611189d..7405e88 100644
Binary files a/bundles/org.eclipse.equinox.p2.ui.discovery/icons/etool16/find.png and b/bundles/org.eclipse.equinox.p2.ui.discovery/icons/etool16/find.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.discovery/icons/etool16/find@2x.png b/bundles/org.eclipse.equinox.p2.ui.discovery/icons/etool16/find@2x.png
new file mode 100644
index 0000000..417cd8e
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.discovery/icons/etool16/find@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/iu_disabled_obj.gif b/bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/iu_disabled_obj.gif
deleted file mode 100644
index f192b84..0000000
Binary files a/bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/iu_disabled_obj.gif and /dev/null differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/iu_disabled_obj.png b/bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/iu_disabled_obj.png
new file mode 100644
index 0000000..2903413
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/iu_disabled_obj.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/iu_disabled_obj@2x.png b/bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/iu_disabled_obj@2x.png
new file mode 100644
index 0000000..fda2009
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/iu_disabled_obj@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/iu_obj.gif b/bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/iu_obj.gif
deleted file mode 100644
index 8706680..0000000
Binary files a/bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/iu_obj.gif and /dev/null differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/iu_obj.png b/bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/iu_obj.png
new file mode 100644
index 0000000..684e82a
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/iu_obj.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/iu_obj@2x.png b/bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/iu_obj@2x.png
new file mode 100644
index 0000000..d61d826
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/iu_obj@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/iu_update_obj.gif b/bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/iu_update_obj.gif
deleted file mode 100644
index 16f4f7c..0000000
Binary files a/bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/iu_update_obj.gif and /dev/null differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/iu_update_obj.png b/bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/iu_update_obj.png
new file mode 100644
index 0000000..a851004
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/iu_update_obj.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/iu_update_obj@2x.png b/bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/iu_update_obj@2x.png
new file mode 100644
index 0000000..f0c1fde
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/iu_update_obj@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/message_info.gif b/bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/message_info.gif
deleted file mode 100644
index b484d9b..0000000
Binary files a/bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/message_info.gif and /dev/null differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/message_info.png b/bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/message_info.png
new file mode 100644
index 0000000..7fe40bb
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/message_info.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/message_info@2x.png b/bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/message_info@2x.png
new file mode 100644
index 0000000..a10214e
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.discovery/icons/obj16/message_info@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.discovery/icons/ovr32/message_warning.png b/bundles/org.eclipse.equinox.p2.ui.discovery/icons/ovr32/message_warning.png
new file mode 100644
index 0000000..7d7fc73
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.discovery/icons/ovr32/message_warning.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.discovery/icons/ovr32/message_warning@2x.png b/bundles/org.eclipse.equinox.p2.ui.discovery/icons/ovr32/message_warning@2x.png
new file mode 100644
index 0000000..a5ced85
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.discovery/icons/ovr32/message_warning@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.discovery/icons/wizban/banner-discovery.png b/bundles/org.eclipse.equinox.p2.ui.discovery/icons/wizban/banner-discovery.png
index cc4b2cf..b8b4c65 100644
Binary files a/bundles/org.eclipse.equinox.p2.ui.discovery/icons/wizban/banner-discovery.png and b/bundles/org.eclipse.equinox.p2.ui.discovery/icons/wizban/banner-discovery.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.discovery/icons/wizban/banner-discovery@2x.png b/bundles/org.eclipse.equinox.p2.ui.discovery/icons/wizban/banner-discovery@2x.png
new file mode 100644
index 0000000..dc32d5f
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.discovery/icons/wizban/banner-discovery@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.discovery/src/org/eclipse/equinox/internal/p2/ui/discovery/DiscoveryImages.java b/bundles/org.eclipse.equinox.p2.ui.discovery/src/org/eclipse/equinox/internal/p2/ui/discovery/DiscoveryImages.java
index fefafbc..0966b93 100644
--- a/bundles/org.eclipse.equinox.p2.ui.discovery/src/org/eclipse/equinox/internal/p2/ui/discovery/DiscoveryImages.java
+++ b/bundles/org.eclipse.equinox.p2.ui.discovery/src/org/eclipse/equinox/internal/p2/ui/discovery/DiscoveryImages.java
@@ -37,17 +37,17 @@ public class DiscoveryImages {
 	/**
 	 * image descriptor for a warning overlay suitable for use with 32x32 images.
 	 */
-	public static final ImageDescriptor OVERLAY_WARNING_32 = create(T_OVR_32, "message_warning.gif"); //$NON-NLS-1$
+	public static final ImageDescriptor OVERLAY_WARNING_32 = create(T_OVR_32, "message_warning.png"); //$NON-NLS-1$
 
 	public static final ImageDescriptor BANNER_DISOVERY = create(T_WIZBAN, "banner-discovery.png"); //$NON-NLS-1$
 
-	public static final ImageDescriptor IU_AVAILABLE = create(T_OBJ_16, "iu_disabled_obj.gif"); //$NON-NLS-1$
+	public static final ImageDescriptor IU_AVAILABLE = create(T_OBJ_16, "iu_disabled_obj.png"); //$NON-NLS-1$
 
-	public static final ImageDescriptor IU_INSTALLED = create(T_OBJ_16, "iu_obj.gif"); //$NON-NLS-1$
+	public static final ImageDescriptor IU_INSTALLED = create(T_OBJ_16, "iu_obj.png"); //$NON-NLS-1$
 
-	public static final ImageDescriptor IU_UPDATABLE = create(T_OBJ_16, "iu_update_obj.gif"); //$NON-NLS-1$
+	public static final ImageDescriptor IU_UPDATABLE = create(T_OBJ_16, "iu_update_obj.png"); //$NON-NLS-1$
 
-	public static final ImageDescriptor MESSAGE_INFO = create(T_OBJ_16, "message_info.gif"); //$NON-NLS-1$
+	public static final ImageDescriptor MESSAGE_INFO = create(T_OBJ_16, "message_info.png"); //$NON-NLS-1$
 
 	public static final ImageDescriptor FIND_CLEAR = create(T_TOOL, "find-clear.gif"); //$NON-NLS-1$
 
diff --git a/bundles/org.eclipse.equinox.p2.ui.importexport/icons/install_wiz.gif b/bundles/org.eclipse.equinox.p2.ui.importexport/icons/install_wiz.gif
deleted file mode 100644
index 7eee7c3..0000000
Binary files a/bundles/org.eclipse.equinox.p2.ui.importexport/icons/install_wiz.gif and /dev/null differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.importexport/icons/wizban/install_wiz.png b/bundles/org.eclipse.equinox.p2.ui.importexport/icons/wizban/install_wiz.png
new file mode 100644
index 0000000..35f23e9
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.importexport/icons/wizban/install_wiz.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.importexport/icons/wizban/install_wiz@2x.png b/bundles/org.eclipse.equinox.p2.ui.importexport/icons/wizban/install_wiz@2x.png
new file mode 100644
index 0000000..c1ef3eb
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.importexport/icons/wizban/install_wiz@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.importexport/plugin.xml b/bundles/org.eclipse.equinox.p2.ui.importexport/plugin.xml
index 7b9ace0..d3b4800 100755
--- a/bundles/org.eclipse.equinox.p2.ui.importexport/plugin.xml
+++ b/bundles/org.eclipse.equinox.p2.ui.importexport/plugin.xml
@@ -10,7 +10,7 @@
       <wizard
             category="org.eclipse.equinox.p2.category"
             class="org.eclipse.equinox.internal.p2.importexport.internal.wizard.ExportWizard"
-            icon="icons/install_wiz.gif"
+            icon="icons/wizban/install_wiz.png"
             id="org.eclipse.equinox.p2.replication.export"
             name="%export.name">
             <description>
@@ -27,17 +27,17 @@
       <wizard
             category="org.eclipse.equinox.p2.category"
             class="org.eclipse.equinox.internal.p2.importexport.internal.wizard.ImportWizard"
-            icon="icons/install_wiz.gif"
+            icon="icons/wizban/install_wiz.png"
             id="org.eclipse.equinox.p2.replication.import"
             name="%import.name">
             <description>
             	%import.description
-            </description>            
+            </description>
       </wizard>
       <wizard
             category="org.eclipse.equinox.p2.category"
             class="org.eclipse.equinox.internal.p2.importexport.internal.wizard.ImportFromInstallationWizard"
-            icon="icons/install_wiz.gif"
+            icon="icons/wizban/install_wiz.png"
             id="org.eclipse.equinox.p2.replication.importfrominstallation"
             name="%importfrominstallation.name">
          <description>
diff --git a/bundles/org.eclipse.equinox.p2.ui.importexport/src/org/eclipse/equinox/internal/p2/importexport/internal/wizard/ExportWizard.java b/bundles/org.eclipse.equinox.p2.ui.importexport/src/org/eclipse/equinox/internal/p2/importexport/internal/wizard/ExportWizard.java
index 86986aa..23799a6 100755
--- a/bundles/org.eclipse.equinox.p2.ui.importexport/src/org/eclipse/equinox/internal/p2/importexport/internal/wizard/ExportWizard.java
+++ b/bundles/org.eclipse.equinox.p2.ui.importexport/src/org/eclipse/equinox/internal/p2/importexport/internal/wizard/ExportWizard.java
@@ -1,10 +1,10 @@
 /*******************************************************************************
  * Copyright (c) 2011 WindRiver Corporation and others.
- * All rights reserved. This program and the accompanying materials 
+ * All rights reserved. This program and the accompanying materials
  * are made available under the terms of the Eclipse Public License v1.0
  * which accompanies this distribution, and is available at
  * http://www.eclipse.org/legal/epl-v10.html
- * 
+ *
  * Contributors:
  *     WindRiver Corporation - initial API and implementation
  *******************************************************************************/
@@ -42,7 +42,7 @@ public class ExportWizard extends AbstractWizard implements IExportWizard {
 
 	public void init(IWorkbench workbench, IStructuredSelection selection) {
 		setWindowTitle(Messages.ExportWizard_WizardTitle);
-		setDefaultPageImageDescriptor(ImageDescriptor.createFromURL(Platform.getBundle(Constants.Bundle_ID).getEntry("icons/install_wiz.gif"))); //$NON-NLS-1$
+		setDefaultPageImageDescriptor(ImageDescriptor.createFromURL(Platform.getBundle(Constants.Bundle_ID).getEntry("icons/wizban/install_wiz.png"))); //$NON-NLS-1$
 		setNeedsProgressMonitor(true);
 	}
 
diff --git a/bundles/org.eclipse.equinox.p2.ui.importexport/src/org/eclipse/equinox/internal/p2/importexport/internal/wizard/ImportFromInstallationWizard.java b/bundles/org.eclipse.equinox.p2.ui.importexport/src/org/eclipse/equinox/internal/p2/importexport/internal/wizard/ImportFromInstallationWizard.java
index 6b89c64..b2e296b 100644
--- a/bundles/org.eclipse.equinox.p2.ui.importexport/src/org/eclipse/equinox/internal/p2/importexport/internal/wizard/ImportFromInstallationWizard.java
+++ b/bundles/org.eclipse.equinox.p2.ui.importexport/src/org/eclipse/equinox/internal/p2/importexport/internal/wizard/ImportFromInstallationWizard.java
@@ -1,62 +1,62 @@
-/*******************************************************************************
- * Copyright (c) 2011 WindRiver Corporation and others.
- * All rights reserved. This program and the accompanying materials 
- * are made available under the terms of the Eclipse Public License v1.0
- * which accompanies this distribution, and is available at
- * http://www.eclipse.org/legal/epl-v10.html
- * 
- * Contributors:
- *     WindRiver Corporation - initial API and implementation
- *******************************************************************************/
-package org.eclipse.equinox.internal.p2.importexport.internal.wizard;
-
-import java.util.Collection;
-import org.eclipse.core.runtime.Platform;
-import org.eclipse.equinox.internal.p2.importexport.internal.*;
-import org.eclipse.equinox.internal.p2.ui.dialogs.ISelectableIUsPage;
-import org.eclipse.equinox.internal.p2.ui.dialogs.InstallWizard;
-import org.eclipse.equinox.internal.p2.ui.model.IUElementListRoot;
-import org.eclipse.equinox.p2.engine.ProvisioningContext;
-import org.eclipse.equinox.p2.metadata.IInstallableUnit;
-import org.eclipse.equinox.p2.operations.InstallOperation;
-import org.eclipse.equinox.p2.ui.LoadMetadataRepositoryJob;
-import org.eclipse.equinox.p2.ui.ProvisioningUI;
-import org.eclipse.jface.dialogs.IDialogSettings;
-import org.eclipse.jface.resource.ImageDescriptor;
-import org.eclipse.jface.viewers.IStructuredSelection;
-import org.eclipse.ui.IImportWizard;
-import org.eclipse.ui.IWorkbench;
-
-public class ImportFromInstallationWizard extends InstallWizard implements IImportWizard {
-
-	public ImportFromInstallationWizard() {
-		this(ProvisioningUI.getDefaultUI(), null, null, null);
-	}
-
-	public ImportFromInstallationWizard(ProvisioningUI ui, InstallOperation operation, Collection<IInstallableUnit> initialSelections, LoadMetadataRepositoryJob preloadJob) {
-		super(ui, operation, initialSelections, preloadJob);
-		IDialogSettings workbenchSettings = ImportExportActivator.getDefault().getDialogSettings();
-		String sectionName = "ImportFromInstallationWizard"; //$NON-NLS-1$
-		IDialogSettings section = workbenchSettings.getSection(sectionName);
-		if (section == null) {
-			section = workbenchSettings.addNewSection(sectionName);
-		}
-		setDialogSettings(section);
-	}
-
-	public void init(IWorkbench workbench, IStructuredSelection selection) {
-		setWindowTitle(Messages.ImportWizard_WINDOWTITLE);
-		setDefaultPageImageDescriptor(ImageDescriptor.createFromURL(Platform.getBundle(Constants.Bundle_ID).getEntry("icons/install_wiz.gif"))); //$NON-NLS-1$
-		setNeedsProgressMonitor(true);
-	}
-
-	@Override
-	protected ISelectableIUsPage createMainPage(IUElementListRoot input, Object[] selections) {
-		return new ImportFromInstallationPage(ui, this);
-	}
-
-	@Override
-	protected ProvisioningContext getProvisioningContext() {
-		return ((ImportFromInstallationPage) mainPage).getProvisioningContext();
-	}
-}
+/*******************************************************************************
+ * Copyright (c) 2011 WindRiver Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     WindRiver Corporation - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.equinox.internal.p2.importexport.internal.wizard;
+
+import java.util.Collection;
+import org.eclipse.core.runtime.Platform;
+import org.eclipse.equinox.internal.p2.importexport.internal.*;
+import org.eclipse.equinox.internal.p2.ui.dialogs.ISelectableIUsPage;
+import org.eclipse.equinox.internal.p2.ui.dialogs.InstallWizard;
+import org.eclipse.equinox.internal.p2.ui.model.IUElementListRoot;
+import org.eclipse.equinox.p2.engine.ProvisioningContext;
+import org.eclipse.equinox.p2.metadata.IInstallableUnit;
+import org.eclipse.equinox.p2.operations.InstallOperation;
+import org.eclipse.equinox.p2.ui.LoadMetadataRepositoryJob;
+import org.eclipse.equinox.p2.ui.ProvisioningUI;
+import org.eclipse.jface.dialogs.IDialogSettings;
+import org.eclipse.jface.resource.ImageDescriptor;
+import org.eclipse.jface.viewers.IStructuredSelection;
+import org.eclipse.ui.IImportWizard;
+import org.eclipse.ui.IWorkbench;
+
+public class ImportFromInstallationWizard extends InstallWizard implements IImportWizard {
+
+	public ImportFromInstallationWizard() {
+		this(ProvisioningUI.getDefaultUI(), null, null, null);
+	}
+
+	public ImportFromInstallationWizard(ProvisioningUI ui, InstallOperation operation, Collection<IInstallableUnit> initialSelections, LoadMetadataRepositoryJob preloadJob) {
+		super(ui, operation, initialSelections, preloadJob);
+		IDialogSettings workbenchSettings = ImportExportActivator.getDefault().getDialogSettings();
+		String sectionName = "ImportFromInstallationWizard"; //$NON-NLS-1$
+		IDialogSettings section = workbenchSettings.getSection(sectionName);
+		if (section == null) {
+			section = workbenchSettings.addNewSection(sectionName);
+		}
+		setDialogSettings(section);
+	}
+
+	public void init(IWorkbench workbench, IStructuredSelection selection) {
+		setWindowTitle(Messages.ImportWizard_WINDOWTITLE);
+		setDefaultPageImageDescriptor(ImageDescriptor.createFromURL(Platform.getBundle(Constants.Bundle_ID).getEntry("icons/wizban/install_wiz.png"))); //$NON-NLS-1$
+		setNeedsProgressMonitor(true);
+	}
+
+	@Override
+	protected ISelectableIUsPage createMainPage(IUElementListRoot input, Object[] selections) {
+		return new ImportFromInstallationPage(ui, this);
+	}
+
+	@Override
+	protected ProvisioningContext getProvisioningContext() {
+		return ((ImportFromInstallationPage) mainPage).getProvisioningContext();
+	}
+}
diff --git a/bundles/org.eclipse.equinox.p2.ui.importexport/src/org/eclipse/equinox/internal/p2/importexport/internal/wizard/ImportWizard.java b/bundles/org.eclipse.equinox.p2.ui.importexport/src/org/eclipse/equinox/internal/p2/importexport/internal/wizard/ImportWizard.java
index f2ed247..7eb1555 100644
--- a/bundles/org.eclipse.equinox.p2.ui.importexport/src/org/eclipse/equinox/internal/p2/importexport/internal/wizard/ImportWizard.java
+++ b/bundles/org.eclipse.equinox.p2.ui.importexport/src/org/eclipse/equinox/internal/p2/importexport/internal/wizard/ImportWizard.java
@@ -57,7 +57,7 @@ public class ImportWizard extends InstallWizard implements IImportWizard {
 
 	public void init(IWorkbench workbench, IStructuredSelection selection) {
 		setWindowTitle(Messages.ImportWizard_WINDOWTITLE);
-		setDefaultPageImageDescriptor(ImageDescriptor.createFromURL(Platform.getBundle(Constants.Bundle_ID).getEntry("icons/install_wiz.gif"))); //$NON-NLS-1$
+		setDefaultPageImageDescriptor(ImageDescriptor.createFromURL(Platform.getBundle(Constants.Bundle_ID).getEntry("icons/wizban/install_wiz.png"))); //$NON-NLS-1$
 		setNeedsProgressMonitor(true);
 	}
 
diff --git a/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/close.gif b/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/close.gif
deleted file mode 100644
index 1aca259..0000000
Binary files a/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/close.gif and /dev/null differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/close.png b/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/close.png
new file mode 100644
index 0000000..478e230
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/close.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/close@2x.png b/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/close@2x.png
new file mode 100644
index 0000000..cf8f6c1
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/close@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/close_hot.gif b/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/close_hot.gif
deleted file mode 100644
index d536016..0000000
Binary files a/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/close_hot.gif and /dev/null differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/close_hot.png b/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/close_hot.png
new file mode 100644
index 0000000..4cc2a86
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/close_hot.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/close_hot@2x.png b/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/close_hot@2x.png
new file mode 100644
index 0000000..49670ae
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/close_hot@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/update.gif b/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/update.gif
deleted file mode 100644
index aa84ebe..0000000
Binary files a/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/update.gif and /dev/null differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/update.png b/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/update.png
new file mode 100644
index 0000000..191d1bd
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/update.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/update@2x.png b/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/update@2x.png
new file mode 100644
index 0000000..544cd5f
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/update@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/update_problems.gif b/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/update_problems.gif
deleted file mode 100644
index c0cfaca..0000000
Binary files a/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/update_problems.gif and /dev/null differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/update_problems.png b/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/update_problems.png
new file mode 100644
index 0000000..dc80915
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/update_problems.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/update_problems@2x.png b/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/update_problems@2x.png
new file mode 100644
index 0000000..bbaaae5
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/icons/tool/update_problems@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/src/org/eclipse/equinox/internal/p2/ui/sdk/scheduler/AutomaticUpdatePlugin.java b/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/src/org/eclipse/equinox/internal/p2/ui/sdk/scheduler/AutomaticUpdatePlugin.java
index f42534d..b50b442 100644
--- a/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/src/org/eclipse/equinox/internal/p2/ui/sdk/scheduler/AutomaticUpdatePlugin.java
+++ b/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/src/org/eclipse/equinox/internal/p2/ui/sdk/scheduler/AutomaticUpdatePlugin.java
@@ -41,10 +41,10 @@ public class AutomaticUpdatePlugin extends AbstractUIPlugin {
 	public final static String ICON_PATH = "$nl$/icons/"; //$NON-NLS-1$
 
 	// tool icons
-	public final static String IMG_TOOL_UPDATE = "tool/update.gif"; //$NON-NLS-1$
-	public final static String IMG_TOOL_UPDATE_PROBLEMS = "tool/update_problems.gif"; //$NON-NLS-1$
-	public final static String IMG_TOOL_CLOSE = "tool/close.gif"; //$NON-NLS-1$
-	public final static String IMG_TOOL_CLOSE_HOT = "tool/close_hot.gif"; //$NON-NLS-1$
+	public final static String IMG_TOOL_UPDATE = "tool/update.png"; //$NON-NLS-1$
+	public final static String IMG_TOOL_UPDATE_PROBLEMS = "tool/update_problems.png"; //$NON-NLS-1$
+	public final static String IMG_TOOL_CLOSE = "tool/close.png"; //$NON-NLS-1$
+	public final static String IMG_TOOL_CLOSE_HOT = "tool/close_hot.png"; //$NON-NLS-1$
 
 	private static AutomaticUpdatePlugin plugin;
 	private static BundleContext context;
@@ -63,7 +63,7 @@ public class AutomaticUpdatePlugin extends AbstractUIPlugin {
 
 	/**
 	 * Returns the singleton plugin instance
-	 * 
+	 *
 	 * @return the instance
 	 */
 	public static AutomaticUpdatePlugin getDefault() {
@@ -76,7 +76,7 @@ public class AutomaticUpdatePlugin extends AbstractUIPlugin {
 
 	/*
 	 * (non-Javadoc)
-	 * 
+	 *
 	 * @see
 	 * org.eclipse.ui.plugin.AbstractUIPlugin#start(org.osgi.framework.BundleContext
 	 * )
@@ -131,7 +131,7 @@ public class AutomaticUpdatePlugin extends AbstractUIPlugin {
 
 	/*
 	 * Overridden to use a profile scoped preference store. (non-Javadoc)
-	 * 
+	 *
 	 * @see org.eclipse.ui.plugin.AbstractUIPlugin#getPreferenceStore()
 	 */
 	public IPreferenceStore getPreferenceStore() {
diff --git a/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/src/org/eclipse/equinox/internal/p2/ui/sdk/scheduler/migration/MigrationWizard.java b/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/src/org/eclipse/equinox/internal/p2/ui/sdk/scheduler/migration/MigrationWizard.java
index a49ac54..ca75cf4 100644
--- a/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/src/org/eclipse/equinox/internal/p2/ui/sdk/scheduler/migration/MigrationWizard.java
+++ b/bundles/org.eclipse.equinox.p2.ui.sdk.scheduler/src/org/eclipse/equinox/internal/p2/ui/sdk/scheduler/migration/MigrationWizard.java
@@ -9,7 +9,7 @@
  *     WindRiver Corporation - initial API and implementation
  *     Ericsson AB - Ongoing development
  *     Ericsson AB (Pascal Rapicault)
- *     Ericsson AB (Hamdan Msheik) 
+ *     Ericsson AB (Hamdan Msheik)
  *     Red Hat Inc. - Bug 460967
  *******************************************************************************/
 package org.eclipse.equinox.internal.p2.ui.sdk.scheduler.migration;
@@ -69,7 +69,7 @@ public class MigrationWizard extends InstallWizard implements IImportWizard {
 
 	public void init(IWorkbench workbench, IStructuredSelection selection) {
 		setWindowTitle(firstTime ? ProvUIMessages.MigrationWizard_WINDOWTITLE_FIRSTRUN : ProvUIMessages.MigrationWizard_WINDOWTITLE);
-		setDefaultPageImageDescriptor(ImageDescriptor.createFromURL(Platform.getBundle(ProvUIActivator.PLUGIN_ID).getEntry("icons/install_wiz.gif"))); //$NON-NLS-1$
+		setDefaultPageImageDescriptor(ImageDescriptor.createFromURL(Platform.getBundle(ProvUIActivator.PLUGIN_ID).getEntry("icons/install_wiz.png"))); //$NON-NLS-1$
 		setNeedsProgressMonitor(true);
 	}
 
diff --git a/bundles/org.eclipse.equinox.p2.ui.sdk/icons/obj/iu_obj.gif b/bundles/org.eclipse.equinox.p2.ui.sdk/icons/obj/iu_obj.gif
deleted file mode 100644
index 8706680..0000000
Binary files a/bundles/org.eclipse.equinox.p2.ui.sdk/icons/obj/iu_obj.gif and /dev/null differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.sdk/icons/obj/iu_obj.png b/bundles/org.eclipse.equinox.p2.ui.sdk/icons/obj/iu_obj.png
new file mode 100644
index 0000000..684e82a
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.sdk/icons/obj/iu_obj.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.sdk/icons/obj/iu_obj@2x.png b/bundles/org.eclipse.equinox.p2.ui.sdk/icons/obj/iu_obj@2x.png
new file mode 100644
index 0000000..d61d826
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.sdk/icons/obj/iu_obj@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.sdk/icons/obj/iu_update_obj.gif b/bundles/org.eclipse.equinox.p2.ui.sdk/icons/obj/iu_update_obj.gif
deleted file mode 100644
index 16f4f7c..0000000
Binary files a/bundles/org.eclipse.equinox.p2.ui.sdk/icons/obj/iu_update_obj.gif and /dev/null differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.sdk/icons/obj/iu_update_obj.png b/bundles/org.eclipse.equinox.p2.ui.sdk/icons/obj/iu_update_obj.png
new file mode 100644
index 0000000..a851004
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.sdk/icons/obj/iu_update_obj.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.sdk/icons/obj/iu_update_obj@2x.png b/bundles/org.eclipse.equinox.p2.ui.sdk/icons/obj/iu_update_obj@2x.png
new file mode 100644
index 0000000..f0c1fde
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.sdk/icons/obj/iu_update_obj@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.sdk/icons/obj/profile_obj.gif b/bundles/org.eclipse.equinox.p2.ui.sdk/icons/obj/profile_obj.gif
deleted file mode 100644
index c984fba..0000000
Binary files a/bundles/org.eclipse.equinox.p2.ui.sdk/icons/obj/profile_obj.gif and /dev/null differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.sdk/icons/obj/profile_obj.png b/bundles/org.eclipse.equinox.p2.ui.sdk/icons/obj/profile_obj.png
new file mode 100644
index 0000000..bbd0d4a
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.sdk/icons/obj/profile_obj.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.sdk/icons/obj/profile_obj@2x.png b/bundles/org.eclipse.equinox.p2.ui.sdk/icons/obj/profile_obj@2x.png
new file mode 100644
index 0000000..b540633
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui.sdk/icons/obj/profile_obj@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui.sdk/plugin.xml b/bundles/org.eclipse.equinox.p2.ui.sdk/plugin.xml
index 3e42b0f..249d783 100644
--- a/bundles/org.eclipse.equinox.p2.ui.sdk/plugin.xml
+++ b/bundles/org.eclipse.equinox.p2.ui.sdk/plugin.xml
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?eclipse version="3.2"?>
 <plugin>
-   	<extension 
+   	<extension
    		id="preferences" point="org.eclipse.core.runtime.preferences" name="%preferences">
 		<initializer class="org.eclipse.equinox.internal.p2.ui.sdk.prefs.PreferenceInitializer"/>
 	</extension>
@@ -13,7 +13,7 @@
             id="org.eclipse.equinox.internal.p2.ui.sdk.ProvisioningPreferencePage">
             <keywordReference id="org.eclipse.equinox.p2.ui.sdk.updates.general"/>
       </page>
-      
+
        <page
             name="%sitesPrefPage"
             category="org.eclipse.equinox.internal.p2.ui.sdk.ProvisioningPreferencePage"
@@ -21,7 +21,7 @@
             id="org.eclipse.equinox.internal.p2.ui.sdk.SitesPreferencePage">
             <keywordReference id="org.eclipse.equinox.p2.ui.sdk.updates.general"/>
       </page>
- 
+
    </extension>
 
       <extension
@@ -30,8 +30,8 @@
             label="%preferenceKeywords.general"
             id="org.eclipse.equinox.p2.ui.sdk.updates.general"/>
       </extension>
-      
-      
+
+
    <extension
          point="org.eclipse.ui.commands">
       <command
@@ -67,9 +67,9 @@
                   commandId="org.eclipse.equinox.p2.ui.sdk.update"
             	  mnemonic="%Update.command.mnemonic"
                   id="org.eclipse.equinox.p2.ui.sdk.update"
-                  icon="icons/obj/iu_update_obj.gif">
+                  icon="icons/obj/iu_update_obj.png">
             </command>
-                        
+
       </menuContribution>
       <menuContribution
             locationURI="menu:help?after=org.eclipse.equinox.p2.ui.sdk.update">
@@ -77,13 +77,13 @@
                   commandId="org.eclipse.equinox.p2.ui.sdk.install"
             	  mnemonic="%Install.command.mnemonic"
                   id="org.eclipse.equinox.p2.ui.sdk.install"
-                  icon="icons/obj/iu_obj.gif">
+                  icon="icons/obj/iu_obj.png">
             </command>
-                        
+
       </menuContribution>
    </extension>
 
-      	<extension 
+      	<extension
    		id="preferences" point="org.eclipse.core.runtime.preferences" name="%preferences">
 		<initializer class="org.eclipse.equinox.internal.p2.ui.sdk.prefs.PreferenceInitializer"/>
 	</extension>
@@ -99,7 +99,7 @@
             class="org.eclipse.equinox.internal.p2.ui.sdk.RevertProfilePageExtensionFactory"
             id="11.org.eclipse.equinox.internal.p2.ui.sdk.RevertProfilePage">
           </page>
-          
+
       </extension>
 
 </plugin>
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/added_overlay.gif b/bundles/org.eclipse.equinox.p2.ui/icons/obj/added_overlay.gif
deleted file mode 100755
index c65d16a..0000000
Binary files a/bundles/org.eclipse.equinox.p2.ui/icons/obj/added_overlay.gif and /dev/null differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/artifact_repo_obj.gif b/bundles/org.eclipse.equinox.p2.ui/icons/obj/artifact_repo_obj.gif
deleted file mode 100644
index 0470e15..0000000
Binary files a/bundles/org.eclipse.equinox.p2.ui/icons/obj/artifact_repo_obj.gif and /dev/null differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/artifact_repo_obj.png b/bundles/org.eclipse.equinox.p2.ui/icons/obj/artifact_repo_obj.png
new file mode 100644
index 0000000..5a4b140
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui/icons/obj/artifact_repo_obj.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/artifact_repo_obj@2x.png b/bundles/org.eclipse.equinox.p2.ui/icons/obj/artifact_repo_obj@2x.png
new file mode 100644
index 0000000..de198ab
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui/icons/obj/artifact_repo_obj@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/category_obj.gif b/bundles/org.eclipse.equinox.p2.ui/icons/obj/category_obj.gif
deleted file mode 100644
index b487d07..0000000
Binary files a/bundles/org.eclipse.equinox.p2.ui/icons/obj/category_obj.gif and /dev/null differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/category_obj.png b/bundles/org.eclipse.equinox.p2.ui/icons/obj/category_obj.png
new file mode 100644
index 0000000..6de0fbd
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui/icons/obj/category_obj.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/category_obj@2x.png b/bundles/org.eclipse.equinox.p2.ui/icons/obj/category_obj@2x.png
new file mode 100644
index 0000000..dc7c144
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui/icons/obj/category_obj@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_add.gif b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_add.gif
deleted file mode 100644
index 252d7eb..0000000
Binary files a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_add.gif and /dev/null differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_add.png b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_add.png
new file mode 100644
index 0000000..a878c3c
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_add.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_add@2x.png b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_add@2x.png
new file mode 100644
index 0000000..96b710c
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_add@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_disabled_obj.gif b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_disabled_obj.gif
deleted file mode 100644
index f192b84..0000000
Binary files a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_disabled_obj.gif and /dev/null differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_disabled_obj.png b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_disabled_obj.png
new file mode 100644
index 0000000..2903413
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_disabled_obj.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_disabled_obj@2x.png b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_disabled_obj@2x.png
new file mode 100644
index 0000000..fda2009
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_disabled_obj@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_disabled_patch_obj.gif b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_disabled_patch_obj.gif
deleted file mode 100644
index 196b83d..0000000
Binary files a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_disabled_patch_obj.gif and /dev/null differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_disabled_patch_obj.png b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_disabled_patch_obj.png
new file mode 100644
index 0000000..dacdf0a
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_disabled_patch_obj.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_disabled_patch_obj@2x.png b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_disabled_patch_obj@2x.png
new file mode 100644
index 0000000..565ee8d
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_disabled_patch_obj@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_downgraded.png b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_downgraded.png
index 533fee8..77746b1 100644
Binary files a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_downgraded.png and b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_downgraded.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_downgraded@2x.png b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_downgraded@2x.png
new file mode 100644
index 0000000..d567529
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_downgraded@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_info.gif b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_info.gif
deleted file mode 100644
index ee2dac4..0000000
Binary files a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_info.gif and /dev/null differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_info.png b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_info.png
new file mode 100644
index 0000000..5294a4f
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_info.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_info@2x.png b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_info@2x.png
new file mode 100644
index 0000000..3d28b21
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_info@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_notadd.gif b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_notadd.gif
deleted file mode 100644
index dfbf68f..0000000
Binary files a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_notadd.gif and /dev/null differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_notadd.png b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_notadd.png
new file mode 100644
index 0000000..74304e6
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_notadd.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_notadd@2x.png b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_notadd@2x.png
new file mode 100644
index 0000000..55043ae
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_notadd@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_obj.gif b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_obj.gif
deleted file mode 100644
index 8706680..0000000
Binary files a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_obj.gif and /dev/null differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_obj.png b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_obj.png
new file mode 100644
index 0000000..684e82a
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_obj.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_obj@2x.png b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_obj@2x.png
new file mode 100644
index 0000000..d61d826
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_obj@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_patch_obj.gif b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_patch_obj.gif
deleted file mode 100755
index 57fc499..0000000
Binary files a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_patch_obj.gif and /dev/null differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_patch_obj.png b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_patch_obj.png
new file mode 100644
index 0000000..c3bd04e
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_patch_obj.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_patch_obj@2x.png b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_patch_obj@2x.png
new file mode 100644
index 0000000..16178fc
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_patch_obj@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_remove.gif b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_remove.gif
deleted file mode 100644
index b6922ac..0000000
Binary files a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_remove.gif and /dev/null differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_remove.png b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_remove.png
new file mode 100644
index 0000000..bcfae20
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_remove.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_remove@2x.png b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_remove@2x.png
new file mode 100644
index 0000000..02a36f1
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_remove@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_update_obj.gif b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_update_obj.gif
deleted file mode 100644
index 16f4f7c..0000000
Binary files a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_update_obj.gif and /dev/null differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_update_obj.png b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_update_obj.png
new file mode 100644
index 0000000..a851004
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_update_obj.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_update_obj@2x.png b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_update_obj@2x.png
new file mode 100644
index 0000000..f0c1fde
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_update_obj@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_upgraded.png b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_upgraded.png
index 4354c6f..55d08a0 100644
Binary files a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_upgraded.png and b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_upgraded.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_upgraded@2x.png b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_upgraded@2x.png
new file mode 100644
index 0000000..e82f5d5
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui/icons/obj/iu_upgraded@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/metadata_repo_obj.gif b/bundles/org.eclipse.equinox.p2.ui/icons/obj/metadata_repo_obj.gif
deleted file mode 100644
index b8568f4..0000000
Binary files a/bundles/org.eclipse.equinox.p2.ui/icons/obj/metadata_repo_obj.gif and /dev/null differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/metadata_repo_obj.png b/bundles/org.eclipse.equinox.p2.ui/icons/obj/metadata_repo_obj.png
new file mode 100644
index 0000000..2cb962a
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui/icons/obj/metadata_repo_obj.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/metadata_repo_obj@2x.png b/bundles/org.eclipse.equinox.p2.ui/icons/obj/metadata_repo_obj@2x.png
new file mode 100644
index 0000000..3227b4f
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui/icons/obj/metadata_repo_obj@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/profile_obj.gif b/bundles/org.eclipse.equinox.p2.ui/icons/obj/profile_obj.gif
deleted file mode 100644
index c984fba..0000000
Binary files a/bundles/org.eclipse.equinox.p2.ui/icons/obj/profile_obj.gif and /dev/null differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/profile_obj.png b/bundles/org.eclipse.equinox.p2.ui/icons/obj/profile_obj.png
new file mode 100644
index 0000000..bbd0d4a
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui/icons/obj/profile_obj.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/profile_obj@2x.png b/bundles/org.eclipse.equinox.p2.ui/icons/obj/profile_obj@2x.png
new file mode 100644
index 0000000..b540633
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui/icons/obj/profile_obj@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/obj/removed_overlay.gif b/bundles/org.eclipse.equinox.p2.ui/icons/obj/removed_overlay.gif
deleted file mode 100755
index 8aaadfb..0000000
Binary files a/bundles/org.eclipse.equinox.p2.ui/icons/obj/removed_overlay.gif and /dev/null differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/ovr/added_overlay.png b/bundles/org.eclipse.equinox.p2.ui/icons/ovr/added_overlay.png
new file mode 100644
index 0000000..e6eeb7e
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui/icons/ovr/added_overlay.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/ovr/added_overlay@2x.png b/bundles/org.eclipse.equinox.p2.ui/icons/ovr/added_overlay@2x.png
new file mode 100644
index 0000000..7b319c2
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui/icons/ovr/added_overlay@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/ovr/removed_overlay.png b/bundles/org.eclipse.equinox.p2.ui/icons/ovr/removed_overlay.png
new file mode 100644
index 0000000..5fd9c70
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui/icons/ovr/removed_overlay.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/ovr/removed_overlay@2x.png b/bundles/org.eclipse.equinox.p2.ui/icons/ovr/removed_overlay@2x.png
new file mode 100644
index 0000000..e123066
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui/icons/ovr/removed_overlay@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/wizban/install_wiz.gif b/bundles/org.eclipse.equinox.p2.ui/icons/wizban/install_wiz.gif
deleted file mode 100644
index 7eee7c3..0000000
Binary files a/bundles/org.eclipse.equinox.p2.ui/icons/wizban/install_wiz.gif and /dev/null differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/wizban/install_wiz.png b/bundles/org.eclipse.equinox.p2.ui/icons/wizban/install_wiz.png
new file mode 100644
index 0000000..35f23e9
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui/icons/wizban/install_wiz.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/wizban/install_wiz@2x.png b/bundles/org.eclipse.equinox.p2.ui/icons/wizban/install_wiz@2x.png
new file mode 100644
index 0000000..c1ef3eb
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui/icons/wizban/install_wiz@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/wizban/revert_wiz.gif b/bundles/org.eclipse.equinox.p2.ui/icons/wizban/revert_wiz.gif
deleted file mode 100644
index 6a19fbf..0000000
Binary files a/bundles/org.eclipse.equinox.p2.ui/icons/wizban/revert_wiz.gif and /dev/null differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/wizban/revert_wiz.png b/bundles/org.eclipse.equinox.p2.ui/icons/wizban/revert_wiz.png
new file mode 100644
index 0000000..9c7e5ae
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui/icons/wizban/revert_wiz.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/wizban/revert_wiz@2x.png b/bundles/org.eclipse.equinox.p2.ui/icons/wizban/revert_wiz@2x.png
new file mode 100644
index 0000000..d24eee3
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui/icons/wizban/revert_wiz@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/wizban/uninstall_wiz.gif b/bundles/org.eclipse.equinox.p2.ui/icons/wizban/uninstall_wiz.gif
deleted file mode 100644
index 1223378..0000000
Binary files a/bundles/org.eclipse.equinox.p2.ui/icons/wizban/uninstall_wiz.gif and /dev/null differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/wizban/uninstall_wiz.png b/bundles/org.eclipse.equinox.p2.ui/icons/wizban/uninstall_wiz.png
new file mode 100644
index 0000000..1df1c7d
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui/icons/wizban/uninstall_wiz.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/wizban/uninstall_wiz@2x.png b/bundles/org.eclipse.equinox.p2.ui/icons/wizban/uninstall_wiz@2x.png
new file mode 100644
index 0000000..933427c
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui/icons/wizban/uninstall_wiz@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/wizban/update_wiz.gif b/bundles/org.eclipse.equinox.p2.ui/icons/wizban/update_wiz.gif
deleted file mode 100644
index 3b36487..0000000
Binary files a/bundles/org.eclipse.equinox.p2.ui/icons/wizban/update_wiz.gif and /dev/null differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/wizban/update_wiz.png b/bundles/org.eclipse.equinox.p2.ui/icons/wizban/update_wiz.png
new file mode 100644
index 0000000..818ee5c
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui/icons/wizban/update_wiz.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/icons/wizban/update_wiz@2x.png b/bundles/org.eclipse.equinox.p2.ui/icons/wizban/update_wiz@2x.png
new file mode 100644
index 0000000..e6c06b7
Binary files /dev/null and b/bundles/org.eclipse.equinox.p2.ui/icons/wizban/update_wiz@2x.png differ
diff --git a/bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/ProvUIImages.java b/bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/ProvUIImages.java
index dd3114d..39a6474 100644
--- a/bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/ProvUIImages.java
+++ b/bundles/org.eclipse.equinox.p2.ui/src/org/eclipse/equinox/internal/p2/ui/ProvUIImages.java
@@ -4,7 +4,7 @@
  *  are made available under the terms of the Eclipse Public License v1.0
  *  which accompanies this distribution, and is available at
  *  http://www.eclipse.org/legal/epl-v10.html
- * 
+ *
  *  Contributors:
  *     IBM Corporation - initial API and implementation
  *     Red Hat, Inc. - support for remediation page
@@ -25,7 +25,7 @@ import org.eclipse.swt.graphics.Image;
  * </p>
  * <p>
  * This class is not intended to be subclassed or instantiated by clients
- * 
+ *
  * @since 3.4
  * @noextend This class is not intended to be subclassed by clients.
  * @noinstantiate This class is not intended to be instantiated by clients.
@@ -35,36 +35,36 @@ public class ProvUIImages {
 	// bundle-relative icon path
 	public final static String ICON_PATH = "$nl$/icons/"; //$NON-NLS-1$
 	//objects
-	public final static String IMG_ARTIFACT_REPOSITORY = "obj/artifact_repo_obj.gif"; //$NON-NLS-1$
-	public final static String IMG_METADATA_REPOSITORY = "obj/metadata_repo_obj.gif"; //$NON-NLS-1$
-	public final static String IMG_IU = "obj/iu_obj.gif"; //$NON-NLS-1$
-	public final static String IMG_DISABLED_IU = "obj/iu_disabled_obj.gif"; //$NON-NLS-1$
-	public final static String IMG_ADDED = "obj/iu_add.gif"; //$NON-NLS-1$
-	public final static String IMG_REMOVED = "obj/iu_remove.gif"; //$NON-NLS-1$
-	public final static String IMG_CHANGED = "obj/iu_update_obj.gif"; //$NON-NLS-1$
-	public final static String IMG_NOTADDED = "obj/iu_notadd.gif"; //$NON-NLS-1$
+	public final static String IMG_ARTIFACT_REPOSITORY = "obj/artifact_repo_obj.png"; //$NON-NLS-1$
+	public final static String IMG_METADATA_REPOSITORY = "obj/metadata_repo_obj.png"; //$NON-NLS-1$
+	public final static String IMG_IU = "obj/iu_obj.png"; //$NON-NLS-1$
+	public final static String IMG_DISABLED_IU = "obj/iu_disabled_obj.png"; //$NON-NLS-1$
+	public final static String IMG_ADDED = "obj/iu_add.png"; //$NON-NLS-1$
+	public final static String IMG_REMOVED = "obj/iu_remove.png"; //$NON-NLS-1$
+	public final static String IMG_CHANGED = "obj/iu_update_obj.png"; //$NON-NLS-1$
+	public final static String IMG_NOTADDED = "obj/iu_notadd.png"; //$NON-NLS-1$
 
-	public final static String IMG_UPDATED_IU = "obj/iu_update_obj.gif"; //$NON-NLS-1$
+	public final static String IMG_UPDATED_IU = "obj/iu_update_obj.png"; //$NON-NLS-1$
 	public final static String IMG_UPGRADED_IU = "obj/iu_upgraded.png"; //$NON-NLS-1$
 	public final static String IMG_DOWNGRADED_IU = "obj/iu_downgraded.png"; //$NON-NLS-1$
-	public final static String IMG_ADDED_OVERLAY = "obj/added_overlay.gif"; //$NON-NLS-1$
-	public final static String IMG_REMOVED_OVERLAY = "obj/removed_overlay.gif"; //$NON-NLS-1$
-	public final static String IMG_PATCH_IU = "obj/iu_patch_obj.gif"; //$NON-NLS-1$
-	public final static String IMG_DISABLED_PATCH_IU = "obj/iu_disabled_patch_obj.gif"; //$NON-NLS-1$
-	public final static String IMG_PROFILE = "obj/profile_obj.gif"; //$NON-NLS-1$
-	public final static String IMG_CATEGORY = "obj/category_obj.gif"; //$NON-NLS-1$
-	public final static String IMG_INFO = "obj/iu_info.gif"; //$NON-NLS-1$
+	public final static String IMG_ADDED_OVERLAY = "ovr/added_overlay.png"; //$NON-NLS-1$
+	public final static String IMG_REMOVED_OVERLAY = "ovr/removed_overlay.png"; //$NON-NLS-1$
+	public final static String IMG_PATCH_IU = "obj/iu_patch_obj.png"; //$NON-NLS-1$
+	public final static String IMG_DISABLED_PATCH_IU = "obj/iu_disabled_patch_obj.png"; //$NON-NLS-1$
+	public final static String IMG_PROFILE = "obj/profile_obj.png"; //$NON-NLS-1$
+	public final static String IMG_CATEGORY = "obj/category_obj.png"; //$NON-NLS-1$
+	public final static String IMG_INFO = "obj/iu_info.png"; //$NON-NLS-1$
 
 	// wizard graphics
-	public final static String WIZARD_BANNER_INSTALL = "wizban/install_wiz.gif"; //$NON-NLS-1$
-	public final static String WIZARD_BANNER_UNINSTALL = "wizban/uninstall_wiz.gif"; //$NON-NLS-1$
-	public final static String WIZARD_BANNER_UPDATE = "wizban/update_wiz.gif"; //$NON-NLS-1$
-	public final static String WIZARD_BANNER_REVERT = "wizban/revert_wiz.gif"; //$NON-NLS-1$
+	public final static String WIZARD_BANNER_INSTALL = "wizban/install_wiz.png"; //$NON-NLS-1$
+	public final static String WIZARD_BANNER_UNINSTALL = "wizban/uninstall_wiz.png"; //$NON-NLS-1$
+	public final static String WIZARD_BANNER_UPDATE = "wizban/update_wiz.png"; //$NON-NLS-1$
+	public final static String WIZARD_BANNER_REVERT = "wizban/revert_wiz.png"; //$NON-NLS-1$
 
 	/**
 	 * Returns the image descriptor for the given image ID. Returns
 	 * <code>null</code> if there is no such image.
-	 * 
+	 *
 	 * @param id
 	 *            the identifier for the image to retrieve
 	 * @return the image descriptor associated with the given ID
@@ -76,7 +76,7 @@ public class ProvUIImages {
 	/**
 	 * Returns the image for the given image ID. Returns <code>null</code> if
 	 * there is no such image.
-	 * 
+	 *
 	 * @param id
 	 *            the identifier for the image to retrieve
 	 * @return the image associated with the given ID. This image is managed in
